e2e_playwright/host_config/host_config_test.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+from playwright.sync_api import Page, expect
+
+from e2e_playwright.conftest import ImageCompareFunction
+
+
+@pytest.fixture(scope="package", autouse=True)
+def apply_restrictive_host_config():
+    """Apply a restrictive host config for all tests in this package."""
+    # We need to do this in a package scope fixture to ensure that its applied
+    # before the app server is started.
+    os.environ["STREAMLIT_SERVER_DISABLE_UNSAFE_HTML_EXECUTION"] = "True"
+    os.environ["STREAMLIT_SERVER_DISABLE_UNSAFE_IFRAMES"] = "True"
+    os.environ[
+        "STREAMLIT_SERVER_DISABLE_ELEMENTS"
+    ] = "bokehChart,cameraInput,downloadButton,fileUploader"
```

Comment:
For this test, do we need all of these? Can't we just do a single screenshot of a single element?

Reply:
Nope, we don't really need more than one element here. But I tried to cover all the elements that are relevant for the SiS config here, which are these four.